From e7c98d50045364e99f137062c068eb1eec26cb27 Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Wed, 2 Jul 2025 12:26:26 +1000
Subject: [PATCH 3/4] platform: raspberrypi: remove unused variable

Fixes this compile error:
platform/raspberry-pi/platform_pi.c:196:9: error: unused variable 'ret' [-Werror=unused-variable]
196 |     int ret = 0;

Change-Id: I2573e85c621b98ab331510ccb13d58db13f17e81
Signed-off-by: Mathew McBride <matt@traverse.com.au>
---
 platform/raspberry-pi/platform_pi.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/platform/raspberry-pi/platform_pi.c b/platform/raspberry-pi/platform_pi.c
index aee573e..93b8bd7 100644
--- a/platform/raspberry-pi/platform_pi.c
+++ b/platform/raspberry-pi/platform_pi.c
@@ -193,8 +193,6 @@ int platform_get_keypassphrase_default(char *password, int vap_index)
 
 int platform_get_ssid_default(char *ssid, int vap_index)
 {
-    int ret = 0;
-
     wifi_hal_dbg_print("%s:%d \n", __func__, __LINE__);
     /* if the vap_index is that of mesh STA or mesh backhaul then try to obtain the ssid from
        /nvram/EasymeshCfg.json file */
-- 
2.39.5

